index.js

```diff
@@ -108,13 +109,17 @@ async function libraryHasChanged(name, libDir, targetDir, hashStore) {
 
 async function findFiles(libDir, targetDir) {
   const ignore = ["**/*", "!node_modules", "!.git"]
+  const { argv: { ignoreGit } } = yargs
+    .alias('i', 'ignoreGit')
+    .describe('ignoreGit', 'ignoring git dependencies');
+
   // TODO: use resolved paths here
   if (targetDir.indexOf(libDir) === 0) {
     // The target dir is in the lib directory, make sure that path is excluded
     ignore.push("!" + targetDir.substr(libDir.length + 1).split(path.sep)[0])
   }
   const files = await globby(ignore, {
-    gitignore: true,
+    gitignore: !ignoreGit,
```

Comment:
If I read  this correctly, I'm confused by the name now, `ignoreGit` is now off by default, which turns the option to globby on. But if you pass the `ignoreGit` flag, the `gitignore` flag will _not_ be applied. Could you either find a better name, or remove the negation and default to true?